stacks-signer/src/signerdb.rs

```diff
@@ -197,7 +221,58 @@ impl SignerDb {
         try_deserialize(result)
     }
 
-    /// Insert a block into the database.
+    /// Return the first signed block in a tenure (identified by its consensus hash)
+    pub fn get_first_signed_block_in_tenure(
```

Comment:
Would be good to add unit tests for all these database accesses similar to the other tables tests.

Reply:
Yep, will add.

Reply:
Added in [cb69287](https://github.com/stacks-network/stacks-core/pull/4956/commits/cb69287b9f20f5fe62453a51db618f56dff31af5)